.github/workflows/checker-framework.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
       - name: Git Diff on fail
         if: failure()
-        run: git diff && git diff > target/checker-${{ matrix.profile }}.patch
+        run: ./.ci/print-diff-as-patch target/checker-${{ matrix.profile }}.patch
```

Comment:
Do we really need patch file creation?
Can we just print to output?

Reply:
> Do we really need patch file creation?

I was thinking of the same thing. I noticed that we are archiving the .patch file for 7 days 

https://github.com/checkstyle/checkstyle/blob/f75123e43c04ea41a37cfc8e8cfe9254baf75088/.github/workflows/checker-framework.yml#L60-L66

Reply:
We can keep a file but we need to mention during failure that it is available, I already forgot about it.

I think that copy-paste command is more easy to use.

Reply:
should we mention about existence of `.patch` file in this .sh file ?

Reply:
if we keep it, yes.

Reply:
done